core/common/src/geometry/GeodeticDatum.ts

```diff
@@ -380,16 +381,16 @@ export class GeodeticTransform implements GeodeticTransformProps {
     }
   }
 
-  /** @internal */
+  /** Creates a Geodetic Transform from JSON representation.
+   * @public */
```

Comment:
GeodeticTransformProps is `@alpha` but this method is `@public`. Run extract-api in imodeljs-common and clean up any errors reported.

Reply:
@pmconne Documentation changes are occluding functional changes (some remaining). I will abandon this pull request and make changes as three distinct pull requests.